### PR TITLE
fix(gh-action): failing versioning gh-action

### PR DESCRIPTION
**Pull request by gkampitakis**

### CODE REVIEW

File: .github/workflows/versioning.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ jobs:
   versioning:
     name: "SemVer"
     runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (hitmands) [inline .github/workflows/versioning.yml]: I don't think this is needed...

File: .github/workflows/versioning.yml [hunk 2]
```diff
@@ -20,7 +22,7 @@ jobs:
 
       - name: Executing Semantic Versioning
         run: |
-          git remote set-url origin https://${{ secrets.HITMANDS_PAT }}@github.com/hitmands/grpc-stub.git
+          git remote set-url origin https://hitmands:$GITHUB_TOKEN@github.com/hitmands/grpc-stub.git
```
Comment (hitmands) [inline .github/workflows/versioning.yml]: this can just be `hitmands:${{ secrets.GITHUB_TOKEN }}` ?

Comment (gkampitakis) [reply .github/workflows/versioning.yml]: I haven't seen this syntax using the secret right away. In all examples, they use the env pattern so I don't know if it's working. https://docs.github.com/en/actions/security-guides/encrypted-secrets. 

I am going to give it a try and see if it works

Comment (hitmands) [reply .github/workflows/versioning.yml]: it works, github does replace every occurrence before creating the job.

**Review by hitmands — APPROVED** [review]

**Pull request merged**